### PR TITLE
FIX tags are not showing up in search results after creation

### PR DESCRIPTION
**Pull request by mkuzdowicz**
Fix for Tags not updating in local cache

- fixing serialisation of thrift Kinesis events
- relevant unit tests added

### CODE REVIEW

File: app/modules/clustersync/SectionSyncUpdateProcessor.scala [hunk 1]
```diff
@@ -7,25 +7,40 @@ import play.api.Logger
...
+import scala.util.{Failure, Success, Try}
+
+object SectionEventDeserialiser {
 
   import services.KinesisRecordPayloadConversions._
 
-  override def process(record: Record) {
-    val sectionEvent = SectionEvent.decode(record)
+  def deserialise(record: Record): Try[SectionEvent] = {
+    val tProto = kinesisRecordAsThriftCompactProtocol(record, stripCompressionByte = true)
```
Comment (akash1810) [inline app/modules/clustersync/SectionSyncUpdateProcessor.scala]: Can we add a comment around why we're setting `stripCompressionByte` to true here?

File: app/modules/clustersync/TagSyncUpdateProcessor.scala [hunk 2]
```diff
@@ -2,30 +2,53 @@ package modules.clustersync
...
 
-object TagSyncUpdateProcessor extends KinesisStreamRecordProcessor {
+import scala.util.{Failure, Success, Try}
+
+object TagEventDeserialiser {
 
   import services.KinesisRecordPayloadConversions._
 
+  def deserialise(record: Record): Try[TagEvent] = {
+    val tProto = kinesisRecordAsThriftCompactProtocol(record, stripCompressionByte = true)
```
Comment (akash1810) [inline app/modules/clustersync/TagSyncUpdateProcessor.scala]: Can we add a comment around why we're setting `stripCompressionByte` to true here?

File: app/modules/clustersync/SectionSyncUpdateProcessor.scala (hunk 1 above)
Comment (sihil) [reply app/modules/clustersync/SectionSyncUpdateProcessor.scala]: There is a test that checks this behaviour which kinda acts as a comment, but can't hurt.

File: app/modules/clustersync/TagSyncUpdateProcessor.scala (hunk 2 above)
Comment (mkuzdowicz) [reply app/modules/clustersync/TagSyncUpdateProcessor.scala]: sure

File: app/modules/clustersync/SectionSyncUpdateProcessor.scala (hunk 1 above)
Comment (mkuzdowicz) [reply app/modules/clustersync/SectionSyncUpdateProcessor.scala]: yeah i can add that, test handles that but not say about context of this work-around as it is in some way a work-around

Comment (mkuzdowicz) [reply app/modules/clustersync/SectionSyncUpdateProcessor.scala]: added

File: app/modules/clustersync/TagSyncUpdateProcessor.scala (hunk 2 above)
Comment (mkuzdowicz) [reply app/modules/clustersync/TagSyncUpdateProcessor.scala]: added

**Pull request merged**